src/index.js

```diff
@@ -166,6 +183,17 @@ const App = () => {
         </a>
       </div>`
 
+  const renderDeletePlaylistButton = () =>
+    playList.length > 0 && html`
+      <div
```

Comment:
As I said [here](https://github.com/RodrigoWebDev/anonymous-youtube-playlist-generator/pull/26#issuecomment-1138601531) to fix the icon alignment, first add the `flex` class to the button `<div>`, for the vertical alignment use the `items-center`. One last thing is change the `<div>` to `<button>` since this is a button. It should be like this

```
<button
        class="${css.dangerButton} cursor-pointer flex items-center"
        id="generated-url-div"
        onClick=${() => openDeletePlaylistPopUp()}
      >
        <img class="w-5 opacity-50" src=${trashIcon} />
        Delete playlist
  </button>
```